doc/lfsc1_5.g4

```diff
@@ -78,7 +78,7 @@ ntype
 // (^ c t)
 
 // Extension /////////////////////
-vtype : '(' 'var' iden ntype ')' ;
+vtype : '(' 'id' iden ntype ')' ;
```

Comment:
`var` is used in our boolean signature, so perhaps we should use a different keyword.
I don't feel strongly about what we use.